client/src/app/pages/dashboard/components/controls-completed/controls-completed.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="root">
```

Comment:
Do you thing that this is good name for component? And for what we need this component?

Reply:
The reason it is called _controls_ is that it is a panel at the top of the tasks, and in the future it may contain a lot of buttons like sorting and filtering.

Reply:
Right now we need it to display title for tasks